File: LinguagemC++/EXERCICIO_SOMA_IMPARES.cpp
#include <bits/stdc++.h>

using namespace std;

int main()
{

    int firstNumber, seccondNumber, change, sumNumbers;
    
    cout << "Digite dois numeros: " << endl;
    cin >> firstNumber >> seccondNumber;
    
    if(firstNumber > seccondNumber)
    {
        change = firstNumber;
        firstNumber = seccondNumber;
        seccondNumber = change;
    }
    
    sumNumbers = 0;
    for (int i = firstNumber+1; i < seccondNumber-1; i++)
    {
        if (i % 2 != 0)
        {
            sumNumbers += i;
        }
    }
    
    cout << "Soma dos impares: " << sumNumbers << endl;
    
    return 0;
}

File: LinguagemPython/exercicioSomaImpares.py
# Lê dois números e imprime a soma dos ímpares entre eles

firstNumber: int
seccondNumber: int
change: int
sumNumbers: int

print("Digite dois números: ")
firstNumber = int(input())
seccondNumber = int(input())

if firstNumber > seccondNumber:
    change = seccondNumber
    seccondNumber = firstNumber
    firstNumber = change

sumNumbers = 0
for i in range(firstNumber + 1, seccondNumber - 1):
    if i % 2 != 0:
        sumNumbers += i

print(f"Soma ímpares: {sumNumbers}")
